tsconfig.json

```diff
@@ -18,5 +18,8 @@
             "@fundamental-ngx/core": ["dist/libs/core",   "libs/core/src/index.ts"],
             "@fundamental-ngx/platform": ["libs/platform/src/index.ts"]
         }
-    }
+    },
+  "angularCompilerOptions": {
+    "enableIvy": false
```

Comment:
this is not enabling it

Reply:
I wanted to disabled IVY in general and enable only for `docs` page.

Reply:
if i want to build the library with IVY then i should set it to true, right?